account_teams_test.go

```diff
@@ -120,8 +121,8 @@ func setupAccountsTeamsTestCase(t *testing.T) (*Client, func(t *testing.T)) {
 	}))
 
 	apiUrl = ts.URL
-
-	c, err := NewUserClient(UserName, UserPassword, "aiven-go-client-test/"+Version())
+	ctx := context.Background()
```

Comment:
is there a way to initialize this globally for each test?

Reply:
Tests must be independent, and might have its very own timeout.